collector/collector.go

```diff
@@ -517,7 +530,8 @@ func printTableEntry(
 
 	if !foreground {
 		logger.Info("Result",
-			zap.String("host", targetHost),
+			zap.String("target", targetHost),
+			zap.String("location", targetLocation),
```

Comment:
I used `target_location` in the metrics to differentiate this from the uploading agent's `location` tag. This is implied to be the target location. I can still rename it though to `target_location`.